api/tests/mocha/controllers/well-known.spec.js

```diff
@@ -0,0 +1,52 @@
+const sinon = require('sinon');
+const chai = require('chai');
+
+const controller = require('../../../src/controllers/well-known');
+const settingsService = require('../../../src/services/settings');
+
+describe('well-known controller', () => {
+  let req;
+  let res;
+  let settingsGet;
+
+  beforeEach(() => {
+    req = {
+      accepts: sinon.stub().returns('json'),
+    };
+    res = {
+      json: sinon.stub(),
+      status: sinon.stub(),
+      type: sinon.stub(),
+      end: sinon.stub(),
```

Comment:
No need to change anything, but I guess you could have avoided stubbing these if you had just stubbed out `serverUtils.error`...

Reply:
Right, I wanted to explicitly assert the HTTP status code sent back in the response but there may be a way to do that while stubbing only `serverUtils.error`
I'll either comment back here or push another commit with just that and resolve this discussion. I think it would be better to mock direct dependencies only and not some deeply nested `require()` dependency